meetings/2018-06-14.md

```diff
@@ -0,0 +1,168 @@
+# Node.js Foundation Community Committee 
+## Meeting 2018-06-14
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=xegrAiHy20g
+* **GitHub Issue**: https://github.com/nodejs/community-committee/issues/335
+* **Minutes Google Doc**: https://docs.google.com/document/d/1QcKlkXSgnELDF6NU4zNnMXqS_TTbRsSnl_NUQaRmLIM/edit
+
+## Present
+
+* @waleedashraf (Waleed Ashraf)
+* @jembijoux (Jem Bezooyen)
+* @Tiriel (Benjamin Zaslavsky - member)
+* @detrohutt (A.J. Roberts)
+* @AnnMarieTonog (Ann Marie Tonog - observer) 
+* @chowdhurian (Manil Chowdhury - Member)
+* @hackygolucky (Tracy Hinds – CommComm Director 😍)
+* @amiller-gh (Adam Miller - Member)
+* @ohoe (Rachel White - Member)
+* @williamkapke (William Kapke - Member)
```

Comment:
I was there and thought I'd added myself to the list, although I did leave 15 mins early
* @mhdawson(Michael Dawson - Member)